src/share/types.hpp

```diff
@@ -369,6 +373,7 @@ class types final {
 
           // Extra
           {"fn", key_code::vk_fn_modifier},
+          {"hyper", key_code::vk_hyper_modifier}, // equal to ctrl + option + cmd
```

Comment:
I think that is a matter of opinion, though. There is no formal definition of a hyper key on macOS. Personally I find it most useful to have *hyper* as `ctrl + option + cmd` (when mapping to caps lock) giving you `hyper` and `hyper + shift` for an even wider space of combinations.